integrations/pytorch/utils.py

```diff
@@ -237,9 +237,9 @@ def create_model(args: Any, num_classes: int) -> Module:
     with torch_distributed_zero_first(args.local_rank):  # only download once locally
         if args.checkpoint_path == "zoo":
             if args.recipe_path and args.recipe_path.startswith("zoo:"):
-                args.checkpoint_path = Zoo.download_recipe_base_framework_files(
-                    args.recipe_path, extensions=[".pth"]
-                )[0]
+                args.checkpoint_path = (
```

Comment:
the previous `download_recipe_base_framework_files` function had logic for sorting out the appropriate checkpoint file (ie `model.ckpt.pth` for transfer learning stubs) - does `training.default` take this into account based on the input stub? either way looks like we need to add some flexibility to find the .ckpt

Reply:
No, we do not have this functionality. But should be quite trivial to implement. On it.